mainnet-images.bzl

```diff
@@ -0,0 +1,74 @@
+"""
+This module defines Bazel targets for the mainnet versions of ICOS images
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+MAINNET_REVISION = "6968299131311c836917f0d16d0b1b963526c9b1"
+
+def base_download_url(git_commit_id, variant, update, test):
+    if test:
```

Comment:
optional nit: test and update could be inlined via ternary expressions (`X if COND else Y`)